foundationdb/src/transaction.rs

```diff
@@ -630,8 +630,10 @@ impl GetResult {
     }
 
     /// Returns the values associated with this get
-    pub fn value(&self) -> Result<Option<&[u8]>> {
-        self.inner.get_value()
+    pub fn value(&self) -> Option<&[u8]> {
+        self.inner
+            .get_value()
+            .expect("inner should resolve into value")
```

Comment:
What are the cases where this could fail? It's a binary slice, so I'm thinking it would be a problem with the underlying C api, right?

Reply:
Yes. fdb-c `fdb_future_get_value` might return error when the future returns a value other then value, for example, key or keyvalues.

https://github.com/bluejekyll/foundationdb-rs/blob/master/foundationdb/src/future.rs#L192-L199

To make sure, we check if it resolves to value type before returning `TrxGet`, so I think those `expect` should not be reachable.

https://github.com/bluejekyll/foundationdb-rs/blob/master/foundationdb/src/future.rs#L192-L199